mappings/net/minecraft/block/BlockState.mapping

```diff
@@ -8,6 +8,7 @@ CLASS net/minecraft/class_2680 net/minecraft/block/BlockState
 		FIELD field_16560 shapes [Lnet/minecraft/class_265;
 		FIELD field_19360 collisionShape Lnet/minecraft/class_265;
 		FIELD field_19429 solidFullSquare [Z
+		FIELD field_20337 isShapeFullCube Z
```

Comment:
please no `is` prefix for fields

Reply:
yeah, prefer names like `fullCubeShape` for fields

Reply:
Fixed